### PR TITLE
Support Thoth ocp4 stage deployment

### PR DESCRIPTION
**Pull request by harshad16**
## This Pull Request implements

Support Thoth ocp4 stage deployment
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

- created a different project `thoth-stage`.
- policies for the project.
- application that would be running in this project. 

## If migrating an Application to ArgoCD
- [x] I have completed the list of action items on [this page](https://aicoe.github.io/aicoe-cd/get_argocd_to_manage_your_app/)

:crossed_fingers:  hopefully everything gets deployed successfully.

### CODE REVIEW

File: manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-qeb-hwt.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  project: thoth-stage
+  source:
+    repoURL: 'https://github.com/thoth-station/thoth-application.git'
+    path: qeb-hwt-github-app/overlays/ocp4-stage
+    targetRevision: HEAD
+  destination:
+    server: 'https://api.ocp4.prod.psi.redhat.com:6443'
+    namespace: thoth-frontend-stage
+  syncPolicy:
+    automated: {}
```
Comment (anishasthana) [inline manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-qeb-hwt.yaml:16]: you don't want pruning in this app?

**Review by anishasthana — APPROVED** [review]
lgtm

File: manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-qeb-hwt.yaml (hunk 1 above)
Comment (harshad16) [reply manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-qeb-hwt.yaml:16]: ah good catch , will update :)

**Comment by anishasthana** [comment]
@harshad16 feel free to remove the hold" once you're ready :-)

**Review by anishasthana — APPROVED** [review]

**Pull request merged**